lib/subprocess.js

```diff
@@ -151,6 +151,9 @@ class SubProcess extends EventEmitter {
           err = formatEnoent(err, this.cmd, this.opts?.cwd);
         }
         reject(err);
+
+        this.proc.unref();
```

Comment:
I'm not completely sure about this one. unrefing the variable would mean we allow the process to run even if the parent one has been completed. Is that what we want?

Reply:
The child process _isn't_ running at this point--it raised an `error` which generally means "failed to spawn" afaict